assets/src/edit-story/components/tablist/panel.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import { Icons } from '../../../design-system';
+import { PANEL_STATES } from './constants';
+import {
+  Badge,
+  ButtonText,
+  IconContainer,
+  PanelWrapper,
+  SmallHeadline,
+  TabButton,
+  TabPanel,
+} from './styles';
+
+const Panel = ({
+  badgeCount = 0,
+  children,
+  className,
+  isExpanded,
+  onClick,
+  status = PANEL_STATES.NORMAL,
+  title,
+}) => {
+  const panelId = useMemo(uuidv4, []);
+
+  return (
+    <PanelWrapper className={className} isExpanded={isExpanded}>
+      <TabButton
+        aria-controls={panelId}
+        aria-selected={isExpanded}
+        onClick={onClick}
+        role="tab"
+        status={status}
+      >
+        <ButtonText>
+          <IconContainer>
+            <Icons.ChevronDownSmall />
+          </IconContainer>
+          <SmallHeadline id={`${title}-${panelId}`}>{title}</SmallHeadline>
```

Comment:
Let's not use the `SmallHeadline` here either (similar comment on the badge count usage). This is making the button have an element inside them that is an h1 tag, which should only be used once on the page 🤪  

The `Text` with size small and `isBold` will do the same design here. 

<img width="575" alt="Screen Shot 2021-06-24 at 10 05 53 AM" src="https://user-images.githubusercontent.com/10720454/123304418-c8623780-d4d3-11eb-8de5-9cb188c52bcc.png">